portfolio/src/main/webapp/script.js

```diff
@@ -13,17 +13,23 @@
 // limitations under the License.
 
 function displayComment(limValue){
-    fetch('/comments?limit='+ limValue).then(response => response.json()).then(data => {
-        const commentContainer = document.getElementById("comment_container");
+    var fetchString = '/comments';
+    if(limValue){
+        fetchString += "?limit=";
+        fetchString += limValue;
+    }
+    commentContainer = document.getElementById("comment_container");
+    commentContainer.innerHTML = "";
```

Comment:
might want to move this to be after the fetch but before the for loop. What happens if our call to fetch fails for some reason? We wouldn't want to remove all comments from the screen ahead of time, so we can conditionally clear them out by moving this to the line below.